### PR TITLE
Connect support

### PR DESCRIPTION
**Pull request by oltarasenko**

### CODE REVIEW

**Comment by redapple** [comment]
`verify='/usr/share/ca-certificates/extra/crawlera-ca.crt'` will only work for non-Windows essentially I think, and if you have the CA cert at that exact location depending on where your system puts it.

I would suggest:
- either bundle the CA certificate with crawlera-tools
- or add a "download CA" script to fetch the certificate locally
- in both cases above, change the `verify='/usr/share/ca-certificates/extra/crawlera-ca.crt'` to some local path, e.g. `verify='./crawlera-ca.crt'`
- or add a `--ca-cert` command line option to point to wherever you have the CA cert

File: crawlera-bench [hunk 1]
```diff
@@ -30,28 +30,29 @@ def worker_request(
...
     while True:
         url = queue.get()
-        netloc = urlparse(url).netloc # TODO: we should get this from crawlera
+        netloc = urlparse(url).netloc  # TODO: we should get this from crawlera
         stats = allstats[netloc]
         t = time.time()
 
         # Exit bench after a given amount of time.
         if run_time is not 0 and t - start_time > run_time:
+            print "Killing the bench", run_time
```
Comment (qrilka) [inline crawlera-bench]: maybe `benchmark`?

Comment (qrilka) [inline crawlera-bench]: and probably we shouldn't use "kill" here

File: crawlera-bench [hunk 2]
```diff
@@ -30,28 +30,29 @@ def worker_request(
...
+            print "Start time", start_time, t-start_time
             os._exit(1)
-        if url.startswith("https:"):
-            url = "http://" + url[8:]
-            headers = {"x-crawlera-use-https": "1"}
-        else:
-            headers = {}
+        headers = {}
```
Comment (qrilka) [inline crawlera-bench]: why do we need `headers` at all?

File: crawlera-bench [hunk 3]
```diff
@@ -30,28 +30,29 @@ def worker_request(
...
-                             auth=auth,
-                             allow_redirects=False)
+            r = requests.get(
+                url,
+                headers=headers,
+                proxies=proxies,
+                timeout=timeout,
+                auth=auth,
+                allow_redirects=False,
+                verify='/usr/share/ca-certificates/extra/crawlera-ca.crt')
```
Comment (qrilka) [inline crawlera-bench]: hardcoded path?

**Comment by qrilka** [comment]
Also I think that unless we decide that `X-Crawlera-Use-HTTPS` should be deprecated both methods of requesting https links should be supported

**Comment by oltarasenko** [comment]
Thanks for all comments! I believe I have fixed all remarks above. (Styling remarks are going to be resolved during refactoring)

**Comment by oltarasenko** [comment]
Fixed last portion of CR comments

**Comment by qrilka** [comment]
LGTM

**Pull request merged**